lib/levels.js

```diff
@@ -188,7 +225,8 @@ module.exports = {
   setLevel,
   isLevelEnabled,
   mappings,
```

Comment:
Why has `levels` been removed? This is a breaking change.

Reply:
They actually wasn't they moved to `lib/constants.js`